roles/flag-lifecycle/all-environments.tf

```diff
@@ -67,6 +70,59 @@ resource launchdarkly_custom_role "variation-manager" {
   }
 }
 
+resource launchdarkly_custom_role "approver" {
```

Comment:
How I have it separated right now, the roles in `all-environments.tf` _must_ be allowed in all environments because the actions require it. 

For example: you can only archive a flag in all environments.

Roles like approver/segment manager exist in per-environment since the actions support being granted in each environment. You can still create a global approver/segment manager with the module with the module as it is today though:

```
module "example" {
  source = "./roles/flag-lifecycle"
  project = {
    key  = "default"
    name = "Default"
  }
 
  environments = {
    "all-envs" = {
    key = "*"
    name = "All Environments"
  }
}
```

We could have them always generate global approvers/segment managers too, I just wanted to point out the module already supports creating them as is.